en/asgardeo/docs/apis/organization-apis/restapis/notification-senders.yaml

```
@@ -0,0 +1,1363 @@
+openapi: 3.0.0
+info:
+  title: Notification Senders API
+  description: This document specifies **Email, SMS and Push notification sender RESTful API** for **Asgardeo** organizations.
+               To access the notification sender organization APIs in Asgardeo, you need to first get an access token from your organization.
+  version: "v1"
+servers:
+  - url: https://api.asgardeo.io/t/{root-organization-name}/o/api/server/v1
+    variables:
+      serverUrl:
+        default: https://api.asgardeo.io/t/{root-organization-name}/o/
+      root-organization-name:
+        default: "{root-organization-name}"
+security:
+  - OAuth2: []
+paths:
+  /notification-senders/email:
+    get:
+      tags:
+        - Email Senders
+      summary: Get a list of email senders
+      description: |
+        This API provides the capability to retrieve the list of email senders. <br>
+        
+        <b>Scope(Permission) required:</b> `internal_notification_senders_view`
```

```suggestion
        <b>Scope(Permission) required:</b> `internal_org_notification_senders_view`
```

Check other API scopes of organization API

---

Fixed with https://github.com/wso2/docs-is/pull/5161/commits/d8a6c9c96654a492b94e6d7fb49f9243c440eafd